academy/xl-run-prod/1-software.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Prepare the software to run
+order: 1
+description: Build and deploy
+tag: deep-dive
+---
+
+# Prepare the software to run
+
+To get to production, the first order of business is to build the binary that the nodes will run. If you used Ignite CLI, then you already did this under the hood with the `ignite chain serve` command.
+
+## Target platforms
+
+Because you are going to run the nodes on different machines, it is possible for them to have different operating systems and CPU types. You need to account for that when building. In particular, the computer on which you build may be entirely different from the one on which you eventually run the binary. In the jargon, you need to specify the target platform(s).
+
+What target platforms are available? Because you built your entire blockchain with Go, you can rely on [Go target platforms](https://gist.github.com/asukakenji/f15ba7e588ac42795f421b48b8aede63) for that. To get the targets specific to your version of Go, just run:
```

Comment:
```suggestion
What target platforms are available? Because you built your entire blockchain with Go, you can rely on [Go target platforms](https://go.dev/doc/install/source#environment) for that. Or a [nicely presented one](https://gist.github.com/asukakenji/f15ba7e588ac42795f421b48b8aede63). To get the targets specific to your version of Go, just run:
```